internal/core/data/v2/application/test_utils.go

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package application
```

Comment:
As @cloudxxx8 pointed out this file needs to end in `_test.go`. How about `testUtiles_test.go`?